src/main/java/frc/robot/subsystems/ConveyorSubsystem.java

```diff
@@ -0,0 +1,115 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANEncoder;
+import com.revrobotics.CANPIDController;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.ControlType;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotContainer;
+
+public class ConveyorSubsystem extends SubsystemBase
+{
+    private CANEncoder encoder;
+    private CANPIDController PIDController;
+    private double P, I, D, Iz, FF, maxOutput, minOutput;
+    private boolean spinning;
+    
+    public ConveyorSubsystem()
+    {
+        CANSparkMax motor = new CANSparkMax(10, MotorType.kBrushless);
```

Comment:
Any object you create should probably be declared in as a private of the class, and be assigned in the constructor, so move motor to a private member of Conveyor subsystem and assign it in the constructor.